plugins/mentions.md

```diff
@@ -184,9 +235,16 @@ tinymce.init({
 
 ### `mentions_select`
 
-This option enables you to provide an element to be presented below a hovered mention on the page. This could include more details about the user.
+This option enables you to provide an element to be presented below a hovered mention on the page. This could include more details about the user. You can either create a custom hover element or use a pre-defined one.
```

Comment:
Again, avoid using "you" here please.
```suggestion
This option enables an element to be presented below a mention when hovered over on the page. This could include more details about the user. A custom hover element can created or a pre-defined can be used instead.
```